R/Sharis/Ameriflux_functions.R

```diff
@@ -0,0 +1,92 @@
+generate_attributes_table <- function(csv_file_path,
```

Comment:
If you'd like you can document what this function does.  Check out the multiline comments (#' is the multiline comment symbol in R) at the top of this function: https://github.com/NCEAS/datamgmt/blob/master/R/guess_member_node.R